test/e2e/tests/tokens/custom-token-add-approve.spec.js

```diff
@@ -103,29 +103,27 @@ describe('Create token, approve token and approve token without gas', function (
           css: '.token-allowance-container__verify-link',
         });
 
-        const modalTitle = await driver.waitForSelector({
+        // Verification modal is opened
+        await driver.waitForSelector({
           text: 'Third-party details',
           tag: 'h5',
         });
 
-        assert.equal(await modalTitle.getText(), 'Third-party details');
-
         await driver.clickElementAndWaitToDisappear({
           text: 'Got it',
           tag: 'button',
         });
+        // back to approval modal
         await driver.clickElement({
           text: 'View details',
           css: '.token-allowance-container__view-details',
         });
 
-        // checks elements on approve token popup
-        const functionType = await driver.findElement({
+        // Validate elements on approve token popup
+        await driver.waitForSelector({
           text: 'Function: Approve',
           tag: 'h6',
         });
-        assert.equal(await functionType.getText(), 'Function: Approve');
```

Comment:
Is this check intended to be deleted? If so, is there a reason?

Reply:
Yes actually, i found it not very useful as the selector is targeting text `'Function: Approve'`, and if it doesn't work, it will fail at the `waitForSelector` 
```
  if (!['visible', 'detached'].includes(state)) {
      throw new Error(`Provided state selector ${state} is not supported`);
    }
```